README.md

```diff
@@ -52,10 +52,41 @@ Ready to quickly deploy the latest version of Azure Service Operator on your Kub
     ```sh
     helm repo add azureserviceoperator https://raw.githubusercontent.com/Azure/azure-service-operator/master/charts
     ```
+3. Create an Azure Service Principal. You'll need this to grant Azure Service Operator permissions to create resources in your subscription.
 
-3. Install the Azure Service Operator on your cluster using the following helm install command.
+    First, set the following environment variables to your Azure Tenant ID and Subscription ID with your values:
+    ```yaml
+    AZURE_TENANT_ID=00000000-0000-0000-0000-000000000000
+    AZUR_SUBSCRIPTION_ID=00000000-0000-0000-0000-000000000000
+    ```
+
+    You can find these values by running the following:
+    ```sh
+    az account show
```

Comment:
while subscription is pretty much a universal concept,  AFAIK, no other cloud provider has this "TenantID" concept. It might be good to add a link to what it is.